### ISSUE TITLE
failed to package rpm

### ISSUE THREAD

**Issue by vdvas** [michaelrsweet/epm#67]
Hi.
I use  docker container image centos:latest to build artefacts rpm packet epm.
My .gitlab-ci.yml file is
```
build_rpm_packet:
  stage: build
  image: centos:latest
  before_script:
    - yum update -y
    - yum groupinstall "Development Tools" -y
  script:
    - cd epm
    - ./configure
    - make
    - make install
    - epm -vvv -f rpm epm --output-dir rpm-distr
  artifacts:
    name: "$CI_COMMIT_SHA"
    paths:
      - epm/rpm-distr/
  when: manual
```

I get error  Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1

<details>
  <summary>Gitlab CI console output</summary>
  $ epm -vvv -f rpm epm --output-dir rpm-distr
  Building target platforms: x86_64
  Building for target x86_64
Processing files: epm-4.4.1-0.x86_64
Provides: epm = 4.4.1-0 epm(x86-64) = 4.4.1-0
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Processing files: epm-documentation-4.4.1-0.x86_64
error: File must begin with "/": NONE/share/doc/epm/COPYING
error: File must begin with "/": NONE/share/doc/epm/README
error: File must begin with "/": NONE/share/doc/epm/epm-book.html

RPM build errors:
    File must begin with "/": NONE/share/doc/epm/COPYING
    File must begin with "/": NONE/share/doc/epm/README
    File must begin with "/": NONE/share/doc/epm/epm-book.html
ESP Package Manager v4.4.1
Copyright 1999-2017 by Michael R Sweet.

EPM is free software and comes with ABSOLUTELY NO WARRANTY; for details
see the GNU General Public License in the file COPYING or at
"http://www.fsf.org/gpl.html".  Report all problems to
"https://github.com/michaelrsweet/epm/issues".

Searching for product information...
Stripping executables in distribution...
/usr/bin/strip epm
/usr/bin/strip epminstall
/usr/bin/strip mkepmlist
Creating RPM distribution...
Creating spec file...
Copying temporary distribution files...
epm -> rpm-distr/buildroot/usr/local/bin/epm...
epminstall -> rpm-distr/buildroot/usr/local/bin/epminstall...
mkepmlist -> rpm-distr/buildroot/usr/local/bin/mkepmlist...
./COPYING -> rpm-distr/buildrootNONE/share/doc/epm/COPYING...
./README.md -> rpm-distr/buildrootNONE/share/doc/epm/README...
./doc/epm-book.html -> rpm-distr/buildrootNONE/share/doc/epm/epm-book.html...
./doc/epm.man -> rpm-distr/buildrootNONE/share/man/man1/epm.1...
./doc/epminstall.man -> rpm-distr/buildrootNONE/share/man/man1/epminstall.1...
./doc/mkepmlist.man -> rpm-distr/buildrootNONE/share/man/man1/mkepmlist.1...
./doc/epm.list.man -> rpm-distr/buildrootNONE/share/man/man5/epm.list.5...
Building RPM binary distribution...
/usr/bin/rpmbuild -bb --buildroot "/builds/root/esp/epm/rpm-distr/buildroot" --target x86_64 rpm-distr/epm.spec
Packaging failed!
ERROR: Job failed: exit code 1
</details>

**Comment by skensell201** [comment]
Have you been able to solve this problem?

**Comment by vdvas** [comment]
No.

**Comment by skensell201** [comment]
maybe there are at least some guesses?

**Comment by michaelrsweet** [comment]
Looks like the configure script is substituting NONE for an unspecified prefix. Not sure what is going wrong there, but will investigate.

**Comment by michaelrsweet** [comment]
[master 4a9c861] Fix prefix from configure script (Issue #67)

**Comment by skensell201** [comment]
Michaelsweet thank you so much! It is was very important for me!
